modules/installation-process.adoc

```diff
@@ -3,41 +3,42 @@
+* To deploy a cluster with the {ai-full}, you must configure the cluster settings by using the link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html-single/assisted_installer_for_openshift_container_platform/index[{ai-full}]. There is no installation program to download and configure. After you complete the cluster configuration settings, you download a discovery ISO and then boot cluster machines with that image. You can install clusters with the {ai-full} on Nutanix, vSphere, and bare metal with full integration, and other platforms without integration. If you install on bare metal, you must provide all of the cluster infrastructure and resources, including the networking, load balancing, storage, and individual cluster machines.
 
-* To deploy clusters with the agent-based installer, you download the link:https://console.redhat.com/openshift/install/metal/agent-based[agent-based installer] first. Then, you configure the cluster and generate a discovery image. You boot cluster machines with the discovery image, which installs an agent that communicates with the installation program and handles the provisioning for you instead of you interacting with the installation program or setting up a provisioner machine yourself. You must provide all of the cluster infrastructure and resources, including the networking, load balancing, storage, and individual cluster machines. This approach is ideal for air-gapped or restricted network environments.
+* To deploy clusters with the Agent-based installer, you must download the link:https://console.redhat.com/openshift/install/metal/agent-based[agent-based installer] first. You can then configure the cluster and generate a discovery image. You boot cluster machines with the discovery image, which installs an agent that communicates with the installation program and handles the provisioning for you instead of you interacting with the installation program or setting up a provisioner machine yourself. You must provide all of the cluster infrastructure and resources, including the networking, load balancing, storage, and individual cluster machines. This approach is ideal for air-gapped or restricted network environments.
```

Comment:
```suggestion
* To deploy clusters with the Agent-based Installer, you must download the link:https://console.redhat.com/openshift/install/metal/agent-based[Agent-based Installer] first. You can then configure the cluster and generate a discovery image. You boot cluster machines with the discovery image, which installs an agent that communicates with the installation program and handles the provisioning for you instead of you interacting with the installation program or setting up a provisioner machine yourself. You must provide all of the cluster infrastructure and resources, including the networking, load balancing, storage, and individual cluster machines. This approach is ideal for disconnected environments.
```

Reply:
Also, I am once again wondering if phrases like "must" and "you can then" are necessary in this descriptive content that explains an example workflow of this install

Reply:
I think the addition of modals adds better structure to the sentences. The sentences seem off without them, as our main intent is to target what our users can do with our products.